app/templates/views/guidance/using-notify/unsubscribe-links.html

```diff
@@ -11,28 +11,21 @@
 
   <h1 class="heading-large">Unsubscribe links</h1>
 
-  <p class="govuk-body">Subscription emails must include an unsubscribe link in the body of the message.</p>
-  <p class="govuk-body">If they do not:</p>
-  <ul class="govuk-list govuk-list--bullet">
-    <li>recipients could report your emails as spam</li>
-    <li>email providers may refuse to deliver your messages</li>
-  </ul>
+  <p class="govuk-body">You must let recipients opt out of subscription emails.</p>
```

Comment:
```suggestion
  <p class="govuk-body">You must let people opt out of receiving subscription emails.</p>
```

Just putting this in as something to consider. It feels right to qualify what we mean by ‘opt out’, but we probably don’t want recipients and receiving together in the same sentence.